docs/source/dev-guide/deep-dive-activation.md

```diff
@@ -0,0 +1,204 @@
+
+`conda` ships _virtual environments_ by design. When you install Anaconda or Miniconda, you obtain
+a `base` environment that is essentially a regular environment with some extra checks. These checks
+have to do with what the `conda` command really is and how it is installed in your system.
+
+```{admonition} Base prefix vs target prefix
+Originally, the base installation for `conda` was called the _root_ environment. Every other
+environment lived under `envs/` in that root environment. The root environment was later renamed to
+_base_, but the code still distinguishes between base and target using the old terminology:
+
+* `context.root_prefix`: the path where the `base` conda installation is located.
+* `context.target_prefix`: the environment `conda` is running a command on. Usually, defaults to the
+  activated environment, unless `-n` (name) or `-p` (prefix) is specified in the command line. Note
+  that if you are operating on the `base` environment, the target prefix can have the same value
+  as the root prefix.
+```
+
+When you type `conda` in your terminal, your shell will try find either:
+* a shell function named `conda`
+* an executable file named `conda` in your `PATH` directories
+
+If your `conda` installation has been properly initialized, it will find the shell function. If not,
+it might find the `conda` executable if it _happens_ to be in `PATH`, but this is most often not the
+case. That's why initialization is there to begin with!
+
+## Conda initialization
+
+Why is initialization needed at all, to begin with? There are several reasons:
+
+* Activation requires interacting with the shell context very closely
+* It does not pollute `PATH` unnecesarily
+* Improves performance in certain operations
+
+The main idea behind initialization is to provide a `conda` shell function that allows the Python
+code to interact with the shell context more intimately. It also allows a cleaner `PATH`
+manipulation and snappier responses in some `conda` commands.
+
+The `conda` shell function is mainly a [forwarder function][conda_shell_function]. It will
+delegate most of the commands to the real `conda` executable driven by the Python library.
+However, it will intercept two very specific subcommands:
+* `conda activate`
+* `conda deactivate`
+
+This interception is needed because (de)activation requires exporting (or unsetting) enviroment
+variables back to the shell session (and not just temporarily in the Python process). This will
+be discussed in the next section.
+
+So how is initialization performed? This is the job of the `conda init` subcommand, driven by
+the `conda.cli.main_init` module, which depends direcly on the `conda.core.initialize` module. Let's
+see how this is implemented.
+
+`conda init` will initialize a shell permanently by writing some shell code in the relevant
+startup scripts of your shell (e.g. `~/.bashrc). This is done through different functions defined
+in `conda.core.initialize`, namely:
```

Comment:
The "`" mark here needs to be moved, to fix formatting (see screenshot below):
![Screen Shot 2022-02-08 at 11 26 12 AM](https://user-images.githubusercontent.com/28930622/153031025-ee1b034c-f9ab-45e5-a106-7e1e62c899f8.png)

Reply:
Oh oops, I see that there just needs to be another "`" added right after "~/.bashrc" (vs moving a misplaced backtick)